src/views/questionView.js

```diff
@@ -14,13 +14,12 @@ export const createQuestionElement = (question) => {
   element.innerHTML = String.raw`
     <h1>${question}</h1>
 
-    <ul id="${ANSWERS_LIST_ID}">
+    <ul id="${ANSWERS_LIST_ID}" class='list_of_answers'>
```

Comment:
You are not using this class anywhere so I don't think it is needed?

Reply:
done